nx-X11/programs/Xserver/randr/rrscreen.c

```diff
@@ -976,13 +976,13 @@ ProcRRSetScreenConfig(ClientPtr client)
         has_rate = FALSE;
     }
 
-#ifndef NXAGENT_SERVER
+#ifndef XSERVER_LACKS_PRIVATES_ABI
     rc = dixLookupDrawable(&pDraw, stuff->drawable, client, 0, DixWriteAccess);
-#else                           /* !defined(NXAGENT_SERVER) */
+#else                           /* !defined(XSERVER_LACKS_PRIVATES_ABI) */
     pDraw =
         SecurityLookupDrawable(stuff->drawable, client, DixWriteAccess);
     rc = pDraw ? Success : BadDrawable;
-#endif                          /* !defined(NXAGENT_SERVER) */
+#endif                          /* !defined(XSERVER_LACKS_PRIVATES_ABI) */
 
     if (rc != Success)
         return rc;
```

Comment:
All Randr files still have `NXAGENT` ifdefs for... well, I'm not sure what to call it. Mostly it's for operations on a linked list, which we don't have support for yet, so `XSERVER_LACKS_LINKED_LIST` might sound appropriate at first. But the linked list is actually part of (newer) Screen structures, so maybe `XSERVER_LACKS_SCREEN_LINKED_LISTS`? Maybe you have a better idea for the name. Semantically it would make sense to include them in this PR, though.

There are also `Screen->isGPU` blocks, but they are so rare that I don't think it makes a lot of sense to assign them a special macro.